pyflow/graphics/window.py

```diff
@@ -89,7 +89,7 @@ def createActions(self):
         self._actNew = QAction(
             "&New",
             statusTip="Create new ipygraph",
-            shortcut="Ctrl+N",
+            shortcut="Ctrl+Shift+N",
```

Comment:
Why this modification ?
```suggestion
            shortcut="Ctrl+N",
```
If this is for creating blocks with Ctrl+N, let's wait the concerned PR to do the modification !